package.json

```diff
@@ -15,7 +15,7 @@
     "crowdin:sync": "docusaurus write-translations && crowdin upload && crowdin download"
   },
   "dependencies": {
-    "@crowdin/cli": "3",
+    "@crowdin/cli": "3.6.4",
```

Comment:
Why don't update `yarn.lock`?

Reply:
I used npm to test the website, I will fix it